aviatrix/resource_aviatrix_splunk_logging.go

```diff
@@ -0,0 +1,146 @@
+package aviatrix
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aviatrix/goaviatrix"
+)
+
+func resourceAviatrixSplunkLogging() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAviatrixSplunkLoggingCreate,
+		Read:   resourceAviatrixSplunkLoggingRead,
+		Update: resourceAviatrixSplunkLoggingUpdate,
+		Delete: resourceAviatrixSplunkLoggingDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"server": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Server IP",
+			},
+			"port": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Description: "Port number",
+			},
+			"cu_output_cfg": {
```

Comment:
Is this `cu` short for `custom`, if so we should just not abbreviate it. Same with `cfg`, let's use the non-abbreviated words as much as we can